### DISCUSSION TITLE
Sync saved items

### DISCUSSION THREAD

**Discussion by tdiam** [spacecowboy/Feeder#127] [Ideas]
The device sync feature currently supports syncing feeds and item read statuses.
Is it on the roadmap to also sync saved items across devices?

If yes, I may be able to draft a PR for this after doing some reading. Would you go about it similarly to the current implementation for the read statuses, or would you prioritize [migrating to UnifiedPush](https://ko-fi.com/spacecowboy/posts)?

**Comment by spacecowboy** [comment]
> The device sync feature currently supports syncing feeds and item read statuses. Is it on the roadmap to also sync saved items across devices?

yes it is for API v2

> If yes, I may be able to draft a PR for this after doing some reading. Would you go about it similarly to the current implementation for the read statuses, or would you prioritize [migrating to UnifiedPush](https://ko-fi.com/spacecowboy/posts)?

UnifiedPush is no longer something I intend to go forward with. Too hard for users to use.

API v2 has a couple of changes intended to sync more data.

I should also publish the server code at some point...

**Comment by tnm-** [comment]
Is it possible somehow to sync with a non-android device? (Browser plug-in, or desktop application on linux, etc.)

**Reply by spacecowboy** [reply]
no sync solution for that exists. i don't use anything like that,  so have no reason to build it
